### ISSUE TITLE
Lost connection to Coopnet

### ISSUE THREAD

**Issue by Lucianoiscool2848** [djoslin0/sm64ex-coop#250]
**Describe the bug**
I can't enter any game on mobile. Private nor public.

**To Reproduce**
1. Go to 'join'
2. Click on 'any type of game'
3. See error

**Expected behavior**
Actually able to connect to a game

**Screenshots**
![Screenshot_2023-10-15-00-34-15-714_com owokitty sm64excoop-edit](https://github.com/djoslin0/sm64ex-coop/assets/90996640/c4b5a0a6-4b97-452e-921a-b9141d516a2e)

**Smartphone (please complete the following information):**
 - Device: Redmi 10
 - OS: Android 13
 - Browser: Google
 - Version: 14.0.5

**Additional context**
The weirdest part yet, is that i have the game on pc as well and it loads up the servers all normal.

**Comment by yoyeet961** [comment]
might be something with how your phone's network stuff works

**Comment by TimeParadoxYT** [comment]
Yeah... This problem is caused by your phone. Check your network settings, try changing your DNS.

**Comment by ohhnoobro** [comment]
And what do I do if that happens on PC?

**Comment by yoyeet961** [comment]
Just try doing what TimeParadoxYT said

**Comment by TimeParadoxYT** [comment]
> And what do I do if that happens on PC?

Check your computers firewall and make sure it isn't interfering with the connection

**Comment by ohhnoobro** [comment]
can you explain to me how to do it? cause ido if I did it right pls

**Comment by TimeParadoxYT** [comment]
I can't explain how to do it because all computers are different, and because I don't use windows. But here is what you have to do, open windows defender and allow sm64ex-coop through the firewall. If you have any other antivirus software installed, disable them and check if that fixes the problem

**Comment by plyrnoob2890** [comment]
> > And what do I do if that happens on PC?
> 
> Check your computers firewall and make sure it isn't interfering with the connection

doesn't work
